sdk/identity/identity/test/manual-integration/AzureFunctions/INSTRUCTIONS.md

```diff
@@ -0,0 +1,116 @@
+# Testing Identity in Azure Functions
+
+## Configuration
+
+Before you begin, you must have the following:
+
+- An Azure account with an active subscription
+- The Azure Functions Core Tools version 3.x
+- Node.js, Active LTS version
+- One of the following tools for creating Azure resources:
+  - Azure CLI version 2.4 or later
+  - Azure PowerShell version 5.0 or later
+
+## Prerequisites
+
+Verify your prerequisites, which depend on whether you are using Azure CLI or Azure PowerShell for creating Azure resources
+
+For Azure CLI,
+
+1. In a terminal or command window, run `func --version` to check that the Azure Functions Core Tools are version 3.x.
+2. Run `az --version` to check that the Azure CLI version is 2.4 or later.
+3. Run `az login` to sign in to Azure and verify an active subscription.
+
+For Azure PowerShell,
+
+1. In a terminal or command window, run `func --version` to check that the Azure Functions Core Tools are version 3.x.
+2. Run `(Get-Module -ListAvailable Az).Version` and verify version 5.0 or later.
+3. Run `Connect-AzAccount` to sign in to Azure and verify an active subscription.
+
+### Run Locally
+
+1. Run the following commands in your console from the root folder `IdentityTest`
+
+```
+cd IdentityTest
+npm install
+npm start
+```
+
+Towards the end of the output, you'll see this:
+
+```
+Functions:
+
+        KeyvaultAuthentication: [GET] http://localhost:7071/api/KeyvaultAuthentication
+```
+
+Note:
+If KeyvaultAuthentication doesn't appear as shown above, you likely started the host from outside the root folder of the project. In that case, use Ctrl+C to stop the host, navigate to the project's root folder, and run the previous command again.
+
+2. Copy the URL of your KeyvaultAuthentication function from this output to a browser and append the query string `?name=<your-name>`, making the full URL like
+
+```
+http://localhost:7071/api/KeyvaultAuthentication
+```
+
+The browser should display a message like:
+
+```
+Successfully authenticated with keyvault
+```
+
+The terminal in which you started your project also shows log output as you make requests.
+
+3. When you're ready, use Ctrl+C and choose y to stop the functions host.
+
+## Deploy to Azure
+
+1. Clone the repository azure-sdk-for-js
```

Comment:
Not your problem to solve but the azure-sdk-for-js repo is kind of big to clone just for the single folder.

Do you know if we can publish this as a release or sample that a user can download from docs.microsoft.com or similar?

Reply:
This is a good question. I am not 100% sure who the intended audience for the manual integration tests are - @sadasant

Reply:
Our target audience is mainly partner teams :)

Reply:
>  customers aren't the real audience.

Well, I see how it is.